doc/user/misc.xml

```diff
@@ -675,4 +675,70 @@ env.Command('directory_build_info',
 
   </section>
 
+    <section>
+        <title>Creating LLVM Compilation Database</title>
+
+        <para>
+
+            LLVM has defined a JSON Compilation Database Format. This file is in common use as input into LLVM tools
+            and many IDE's and editors as well.
+        </para>
+        <para>
+            See
+            <ulink url="https://clang.llvm.org/docs/JSONCompilationDatabase.html">
+                <citetitle>JSON Compilation Database Format Specification¶</citetitle>
+            </ulink>
+            for complete information
+        </para>
+
+        <para>
+
+            Currently SCons supports two variations. One is to output source and target files with paths relative to the
+            top
+            of the SCons build, or to output those files with their absolute path.
```

Comment:
Usage mismatch: _two variations. One is...  or to..._.   I'd express this as: "&SCons; can output source and target files either with paths relative to the top of the build, or with absolute paths". The next para can be combined into this one as it's the same concept being described...

Reply:
Or
> The compilation database can be populated with source and target files either either using paths relative to the top of the build, or using absolute paths